apps/files/src/views/FilesAppSettings.vue

```diff
@@ -10,75 +10,63 @@
 		@update:open="onClose">
 		<!-- Settings API-->
 		<NcAppSettingsSection id="settings" :name="t('files', 'General')">
-			<fieldset
+			<NcFormBox>
+				<NcFormBoxSwitch
+					v-model="userConfig.sort_favorites_first"
+					:label="t('files', 'Sort favorites first')"
+					data-cy-files-settings-setting="sort_favorites_first"
+					@update:modelValue="setConfig('sort_favorites_first', $event)" />
```

Comment:
```suggestion
					@update:model-value="setConfig('sort_favorites_first', $event)" />
```

Even though I get warnings like `   18:6  warning  v-on event '@update:modelValue' must be hyphenated  vue/v-on-event-hyphenation`

The hyphenated version does not work.